app/javascript/src/utils/compiler/variables.js

```diff
@@ -1,5 +1,5 @@
 import { findRangesOfStrings, getClosingBracket, matchAllOutsideRanges, splitArgumentsString } from "../parse"
-
+import { evaluateParameterObjects } from "./parameterObjects"
 // NOTE: The fact variable names can start with a decimal is intention.
```

Comment:
For formatting purposes

```suggestion
import { evaluateParameterObjects } from "./parameterObjects"

// NOTE: The fact variable names can start with a decimal is intention.
```